samples/samples.json

```diff
@@ -652,10 +652,10 @@
                 ]
             },
             {
-                "title": "Extending Dash.js",
-                "description": "This sample shows how to use dash.js extend mechanism to add custom HTTP headers and modify URL's of the requests done by the player.",
-                "href": "advanced/extend.html",
```

Comment:
Please also remove `modify-segment-response.html` as we removed `SegmentResponseModifier`

Reply:
Done